app/src/main/java/ch/hikemate/app/utils/DatetimeUtils.kt

```diff
@@ -84,8 +83,3 @@ fun Timestamp.humanReadablePlannedLabel(
     else -> context.getString(R.string.datetime_utils_planned_on_date, formattedDate)
   }
 }
-
-fun Timestamp.toFormattedString(locale: Locale = Locale.getDefault()): String {
```

Comment:
Question: IK its not really part of this PR specifically, but does `toHumanReadableFormat` also work for locales like Chinese and stuff? Could you help me understand what is going on in `humanReadableFormat`, because for me it low-key looks the same as the `toFormattedString` function

Reply:
It is in fact the same as `toFormattedString`, the format being the main difference. `humanReadableFormat` uses `d MMMM yyyy` as format, which will produce dates such as (in English)

- 1 January 2001
- 17 April 2013

It spells out the complete name of the month. Of course, the format itself is not adapted to the locale, but the month is clearly spelled out instead of a number.

`toFormattedString` uses `dd/mm/yyyy`, which can easily be confused with `mm/dd/yyyy`, so I just thought it might be a good idea to use `humanReadableFormat` as well.

However I'm not sure how well it would work with Chinese locale, never tested it

Reply:
ok, thats fine by me. Thanks!